api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -50,6 +50,10 @@ default MigrateTable dropBackup() {
     throw new UnsupportedOperationException("Dropping a backup is not supported");
   }
 
+  default MigrateTable withBackupTableName(String tableName) {
```

Comment:
I think the doc is required.

Reply:
Thank you. Once the commit is merged, I will add the doc. Or if I should add the doc along with this commit please let me know.

Reply:
I mean the document for this method.

Reply:
Sure, let me add it to the method.

Reply:
Is there something I need to add? @ConeyLiu

Reply:
Can we add the new property to `spark-procedures.md` to document it?

Reply:
Of course, sure. I add a new row about the `backup_table_name` argument as a draft. If there's something I need to add more or change, please let me know.